hishel/_async/_storages.py

```diff
@@ -67,13 +67,17 @@ class AsyncFileStorage(AsyncBaseStorage):
     :type base_path: tp.Optional[Path], optional
     :param ttl: Specifies the maximum number of seconds that the response can be cached, defaults to None
     :type ttl: tp.Optional[tp.Union[int, float]], optional
+    :param check_ttl_every: How often in seconds to check staleness of **all** cache files.
+        Makes sense only with set `ttl`, defaults to 60
+    :type check_ttl_every: tp.Union[int, float]
     """
 
     def __init__(
         self,
         serializer: tp.Optional[BaseSerializer] = None,
         base_path: tp.Optional[Path] = None,
         ttl: tp.Optional[tp.Union[int, float]] = None,
+        check_ttl_every: tp.Union[int, float] = 60,
```

Comment:
We can rename this to `vaccum_cache_every` or `clean_cache_every`

Reply:
I  was thinking about `clean_cache_every`. But I think it might be misunderstood with clean all cache every n-seconds. Not just stale.

Reply:
Yeah, true, let's stick with the existing one